
Turn 33
================================================================
warning: owned entities 3, 5 collided on cell (12, 17) as the result of moves on this turn
At command 2 of 5:
    1   m 0 o
>>> 2   m 3 n
    3   m 5 o
    4   m 7 o


Turn 34
================================================================
error: failed to decode bot message: "T" (at position 1)
Last input received was:
Traceback (most recent call last):
  File "MyBot.py", line 106, in <module>
    if can_move_off(me.get_ship(ship_id)) and len(ship_paths[ship_id]) > 0:
  File "C:\Users\justi\Documents\GitHub\ais-halite\hlt\player.py", line 22, in get_ship
    return self._ships[ship_id]
KeyError: 3

